game/game_controller.rb

```diff
@@ -53,6 +54,14 @@ def active_player
     end
   end
 
+  def inactive_player
```

Comment:
You have a attr_accessor defined for `inactive_player` on line six. Does this function override that?